### DISCUSSION TITLE
Groupchat for aTox?

### DISCUSSION THREAD

**Discussion by CTSB99** [evilcorpltd/aTox#1033] [General]
Hey I am trying to create a groupchat for 8 mobile devices with aTox, I can't find any option to create a group in the app, so I installed qTox on my PC and invited the phones to a new group, but I see no option to accept the invite.

**Comment by robinlinden** [comment]
Hi!

Yeah, aTox doesn't yet implement groupchat support. :x It's work in progress, but it needs changes in libraries aTox depends on and I've been very busy, so progress hasn't been very fast.

This feature is being tracked in https://github.com/evilcorpltd/aTox/issues/157

**Reply by CTSB99** [reply]
Thanks
